.github/workflows/golangci-lint.yml

```diff
@@ -4,7 +4,19 @@ on:
     branches:
       - master
       - main
+    paths:
+      - 'operator/**'
+      - 'aggregator/**'
+      - 'common/**'
+      - 'core/**'
+      - '.github/workflows/golangci-lint.yml'
   pull_request:
+    paths:
```

Comment:
Missing:
- metrics

Reply:
Solved in [20793fe](https://github.com/yetanotherco/aligned_layer/pull/763/commits/20793fe4eef1b7d6eb1b59ad662cfae1a6938925)